internal/server/user/user.go

```diff
@@ -27,6 +27,9 @@ type Info struct {
 	// ServiceAccountsByNamespace is the mapping of namespace names to sets of
 	// ServiceAccounts that a user has been mapped to.
 	ServiceAccountsByNamespace map[string]map[types.NamespacedName]struct{}
+	// Username is the username of the user. This is typically the email address
+	// of the user, but may be different depending on the identity provider.
```

Comment:
Small suggestion on wording here, just because it feels like this is more a function of configuration than which identity provider is in use:

```suggestion
	// Username is the username of the user. This is often the email address
	// of the user, but may be different depending on configuration.
```